ledger/sandbox-on-x/reference.conf

```diff
@@ -504,6 +504,12 @@ ledger {
         # is needed to prevent excessive memory pressure.
         max-input-buffer-size = 50
 
+        # The parallelism factor for the Package Metadata In-memory view loading pipeline.
+        package-metadata-view-load-parallelism = 16
```

Comment:
I am wondering whether it would make sense to use a little bit more explanation for this one and/or an improved name, so it get more obvious, that this both is about initialization. WDYT?